crypto/cryptlib.c

```diff
@@ -281,22 +322,27 @@ void OPENSSL_showfatal(const char *fmta, ...)
 
             if (!ReportEvent(hEventLog, EVENTLOG_ERROR_TYPE, 0, 0, NULL,
                              1, 0, &pmsg, NULL)) {
-#if defined(DEBUG)
+#   if! defined(NDEBUG)
```

Comment:
That's an odd formatting? Strangely enough it seems to work at least with gcc and cl. Even if!defined(NDEBUG) works, i.e. without any spaces... Well, even if it works, please fix is :-)